### PR TITLE
Bug 1313317 - History list overlaps the spinner when pull to refresh …

### PR DESCRIPTION
**Pull request by kesun421**
…then scroll down.

*  Corrected the refresh control assignment to the tableview controller.

Corrected refresh control behavior can be seen here: https://youtu.be/5wgEyUtSin8

### CODE REVIEW

**Comment by farhanpatel** [comment]
Thank you for this quick patch! this was quiet the eyesore before. 

I'll make sure this lands in v10.

File: Client/Frontend/Home/HistoryPanel.swift [hunk 1]
```diff
@@ -170,7 +170,7 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         let refresh = UIRefreshControl()
         refresh.addTarget(self, action: #selector(HistoryPanel.refresh), for: UIControlEvents.valueChanged)
         self.refreshControl = refresh
-        self.tableView.addSubview(refresh)
+        self.tableView.refreshControl = refresh
```
Comment (farhanpatel) [inline Client/Frontend/Home/HistoryPanel.swift]: we should make sure that `removeRefreshControl` works correctly. Right now it is doing 
`        self.refreshControl?.removeFromSuperview()`
it probably needs to also do
`self.table.refreshControl = nil`

Comment (kesun421) [reply Client/Frontend/Home/HistoryPanel.swift]: Thanks for pointing that out.  Please review again.

**Comment by farhanpatel** [comment]
This is great! thank you 😄

**Pull request merged**